src/hott/04-half-adjoint-equivalences.rzk.md

```diff
@@ -307,55 +307,55 @@ This will replace the discarded homotopy.
   :=
     \ b →
       concat B
-        ( (section-composite-has-inverse A B f has-inverse-f) b)
-        ( (section-composite-has-inverse A B f has-inverse-f)
-          ((section-composite-has-inverse A B f has-inverse-f) b))
+        ( ( section-composite-has-inverse A B f has-inverse-f) b)
+        ( ( section-composite-has-inverse A B f has-inverse-f)
+          ( ( section-composite-has-inverse A B f has-inverse-f) b))
         ( b)
         ( rev B
-          ( (section-composite-has-inverse A B f has-inverse-f)
-            ((section-composite-has-inverse A B f has-inverse-f) b))
-          ( (section-composite-has-inverse A B f has-inverse-f) b)
+          ( ( section-composite-has-inverse A B f has-inverse-f)
+            ( ( section-composite-has-inverse A B f has-inverse-f) b))
+          ( ( section-composite-has-inverse A B f has-inverse-f) b)
           ( has-inverse-discarded-htpy A B f has-inverse-f
-            ((section-composite-has-inverse A B f has-inverse-f) b)))
+            ( ( section-composite-has-inverse A B f has-inverse-f) b)))
         ( concat B
-          ( (section-composite-has-inverse A B f has-inverse-f)
-            ((section-composite-has-inverse A B f has-inverse-f) b))
-          ( (section-composite-has-inverse A B f has-inverse-f) b)
+          ( ( section-composite-has-inverse A B f has-inverse-f)
+            ( ( section-composite-has-inverse A B f has-inverse-f) b))
+          ( ( section-composite-has-inverse A B f has-inverse-f) b)
           ( b)
           ( ap A B
-            ( (retraction-composite-has-inverse A B f has-inverse-f)
-              (map-inverse-has-inverse A B f has-inverse-f b))
+            ( ( retraction-composite-has-inverse A B f has-inverse-f)
+              ( map-inverse-has-inverse A B f has-inverse-f b))
             ( map-inverse-has-inverse A B f has-inverse-f b) f
-            ( (first (second has-inverse-f))
-              (map-inverse-has-inverse A B f has-inverse-f b)))
-          ( (has-inverse-discarded-htpy A B f has-inverse-f b)))
+            ( ( first (second has-inverse-f))
+              ( map-inverse-has-inverse A B f has-inverse-f b)))
+          ( ( has-inverse-discarded-htpy A B f has-inverse-f b)))
```

Comment:
This looks like a good test case because it incorporates a lot of different indentation levels:
1. indentation starts on the same level as `:=`
2. lambda indentation
3. on the (second to) top level, arguments have the same indentation level as the function they are applied to

```suggestion
  :=
  \ b →
  concat B
  ( ( section-composite-has-inverse A B f has-inverse-f) b)
  ( ( section-composite-has-inverse A B f has-inverse-f)
    ( ( section-composite-has-inverse A B f has-inverse-f) b))
  ( b)
  ( rev B
    ( ( section-composite-has-inverse A B f has-inverse-f)
      ( ( section-composite-has-inverse A B f has-inverse-f) b))
    ( ( section-composite-has-inverse A B f has-inverse-f) b)
    ( has-inverse-discarded-htpy A B f has-inverse-f
      ( ( section-composite-has-inverse A B f has-inverse-f) b)))
  ( concat B
    ( ( section-composite-has-inverse A B f has-inverse-f)
      ( ( section-composite-has-inverse A B f has-inverse-f) b))
    ( ( section-composite-has-inverse A B f has-inverse-f) b)
    ( b)
    ( ap A B
      ( ( retraction-composite-has-inverse A B f has-inverse-f)
        ( map-inverse-has-inverse A B f has-inverse-f b))
      ( map-inverse-has-inverse A B f has-inverse-f b) f
      ( ( first (second has-inverse-f))
        ( map-inverse-has-inverse A B f has-inverse-f b)))
    ( ( has-inverse-discarded-htpy A B f has-inverse-f b)))
```